src/containers/BookList.js

```diff
@@ -18,7 +23,7 @@ const BookList = ({ books, removeBook }) => (
       </thead>
       <tbody>
         {
-          books.map(book => (
+          filteredBooks(filter, books).map(book => (
             <Book
               key={book.bookId}
               title={book.title}
```

Comment:
- We need to make sure that the books are rendered with a conditional statement as stated in [in the Pathwirght to this project](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896829/)
![image](https://user-images.githubusercontent.com/46115725/72271530-abda8400-35ec-11ea-9bcf-5d8ef1e42da7.png)